auth_service/auth_adapter/core/auth.py

```diff
@@ -57,24 +82,14 @@ def __init__(self, config: Config = CONFIG) -> None:
 
         external_keys = config.auth_ext_keys
         if external_keys:
-            try:
-                self.external_jwks = jwk.JWKSet.from_json(external_keys)
-            except Exception as exc:  # pylint:disable=broad-except
-                # do not throw an error so that the auth adapter can still run
-                # even though it will not be able to validate external tokens
-                log.error("Cannot parse external signing keys: %s", exc)
+            self.external_jwks = jwk.JWKSet.from_json(external_keys)
         else:
-            log.error("No external signing keys configured.")
+            raise ConfigurationMissingKey("No external signing keys configured.")
```

Comment:
Maybe it can be simplified into:
```
if not external_keys:
    raise ConfigurationMissingKey("No external signing keys configured.")

# then the else part:
self.external_jwks = jwk.JWKSet.from_json(external_keys)
```

Reply:
Yes, that's better, have changed it.